tornado/httputil.py

```diff
@@ -243,6 +244,86 @@ def __deepcopy__(self, memo_dict):
         return self.copy()
 
 
+_cookie_value_decode = Cookie.SimpleCookie().value_decode
+_cookie_attr_reserved = Cookie.Morsel._reserved
+_cookie_attr_flags = set(["secure", "httponly"])
+
+# part of Cookie library module
+# https://docs.python.org/2.7/library/cookie.html
```

Comment:
Cookie stdlib has no difference between python2.7 and python 3.4, and follow RFC 2109 and RFC 2068.